apps/stats/stats-1.py

```diff
@@ -439,11 +442,14 @@ def __frames( self, script, args ) :
 
 	def __writeScene( self, script, args ) :
 
+		import GafferDispatch
 		import GafferScene
 		import GafferSceneTest
 
 		scene = script.descendant( args["scene"].value )
-		if isinstance( scene, Gaffer.Node ) :
+		if isinstance( scene, GafferScene.Render ) :
+			scene = scene["task"]
```

Comment:
should we be hardcoding the plugname here? could a derived class have multiple out tasks that we might want to profile?

Reply:
We should be, yes. [TaskNodes don't currently support multiple outputs](https://github.com/GafferHQ/gaffer/blob/master/include/GafferDispatch/TaskNode.h#L213-L215), but when they do, we will still want to default to the primary one if only the node is specified. And in the unlikely event of a Render subclass evenutally having more than one task output, you can do `-scene TheRenderNode.theTaskPlug`.